update windows instructions

phaustin opened the pull request:
deleted the bottom of the windows howto -- suggest we also do the following:

1) instead/in addition to sending windows users to the github action output, describe what doesn't currently work, which is I think only that you can't put a url link in the table of contents (i.e. https://github.com/executablebooks/jupyter-book/blob/master/tests/test_tocdirective.py#L113)

2) instead of sending windows users to #906 -- (which turned into a conda tutorial), close and open a new issue about the 3.8 event loop

phaustin commented:
On the event loop issue -- it look like tornado has closed this, but it's still being referenced downstream:
https://github.com/tornadoweb/tornado/issues/2608  

When I run the tests for windows python 3.8, the docs do build (which is what I mainly care about).  The six failing tests look like they're exercising more than one bug though:

https://gist.github.com/phaustin/c98d68ee980ea5619772fb2994ac086d

chrisjsewell commented:
>  it look like tornado has closed this

Well it was closed in June 2019, but only because a workaround was given "Note that it is still necessary to use `asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())` for this platform/version.".
There is still the open issue on nbclient: https://github.com/jupyter/nbclient/issues/85, so I'm suprised it works for you 🤷 

No idea about the DLL load error, what version of jupyter_client are you using.

Note, I generally advocate for running the tests via tox now, to ensure environment consistency (see https://jupyterbook.org/contribute/intro.html#run-the-tests).

chrisjsewell commented on CHANGELOG.md:
```diff
@@ -41,7 +41,7 @@ Here's the headlines:
 Windows support
 : Continuous Integration (CI) testing is now run against Windows OS throughout the EBP stack.
-  The fixes this entailed, mean that Jupyter Book can now be run on Windows with minimal issue (see [Working on Windows](docs/advanced/advanced.md))
+  The fixes this entailed, mean that Jupyter Book can now be run on Windows with minimal issue (see [Working on Windows](docs/advanced/advanced.md#working-on-windows))
```
To the best of my knowledge, this doesn't work with myst, see https://github.com/executablebooks/MyST-Parser/issues/243

phaustin commented:
reverted the CHANGELOG edit and moved the eventloop reference to point to that nbclient issue.   As I mentioned on 
https://github.com/executablebooks/jupyter-book/issues/928#issuecomment-685849562 it looks like conda introduces some differences compared to the github environment -- for the record, I am able to build the docs cleanly with python 3.8:
https://gist.github.com/phaustin/8faf5f791cde5bb2d5a78042acf906f7#file-jb_build38_win-txt
but there are various issues that can wait until we can conda install jupyter-book:
https://gist.github.com/phaustin/8faf5f791cde5bb2d5a78042acf906f7#file-conda_tox38-txt

choldgraf commented:
Hey all - what is the status of this PR? Is it blocking on some other development in the stack? Waiting for review? Needs more work?

mmcky commented:
hey @choldgraf I don't currently have access to a windows machine perhaps we need to get one up and running for testing.

choldgraf commented:
I updated this so that the content was moved to the new windows section, and assuming that tests are happy, I'm gonna merge it in because I think it's an improvement on what we've got for windows thus far. Thanks @phaustin for keeping tabs on this workflow!
